### PR TITLE
Fixing Firefox conflict

### LINKED ISSUE #100: Firefox permission problem due to conflicting package
Firefox is not starting up due to conflicting package in meilix-default-setting
screenshot_firefox_2017-07-22_03 08 49

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
Fixes #100 Firefox still have conflict packages. This PR will remove apport folder and its content and this folder is causing the conflict. 
I have edited only one file and not rebuild the metapackages to see that debuild.sh will work or not.
### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: meilix-default-settings/usr/share/applications/firefox.desktop [hunk 1]
```diff
@@ -1,192 +0,0 @@
-[Desktop Entry]
```
Comment (xeon-zolt) [inline meilix-default-settings/usr/share/applications/firefox.desktop]: i dont think that this file can be a reason of conflict

Comment (meets2tarun) [inline meilix-default-settings/usr/share/applications/firefox.desktop]: It is as it shows the error in the terminal

**Comment by meets2tarun** [comment]
ISO for testing: https://github.com/meets2tarun/meilix/releases/tag/untagged-8ddea6beda8eddb1d88d

File: meilix-default-settings/usr/share/applications/firefox.desktop (hunk 1 above)
Comment (xeon-zolt) [inline meilix-default-settings/usr/share/applications/firefox.desktop]: this is strange

**Pull request merged**